src/modules/expense/Form.tsx

```diff
@@ -0,0 +1,524 @@
+    setSelectedEndAfter(event.target.value as string);
+  };
+
+  const handleFirstStepNextButton = () => {
+    setToggleContinueButton(false);
+    setShowRepeatDetails(true);
+  };
+
+  const handleSecondStepNextButton = () => {
+    setShowRepeatDetails(false);
+    //New continue button and summarize component will be added here.
+    setShowSummarizeSection(true);
+  };
+
+  const handleRepeatSectionContinueButton = () => {
+    handlePopupOpen();
+  };
+
+  const handleEditButton = () => {
+    setShowRepeatDetails(true);
+    setShowSummarizeSection(false);
+  };
+
+  return (
+    <StyledForm>
+      <FormControl fullWidth>
+        <InputLabel id='select-category'>Category</InputLabel>
+        <Select
+          labelId='select-category'
+          id='simple-select-category'
+          multiple
+          value={category}
+          label='Category'
+          onChange={handleChangeCategory}
+          input={<OutlinedInput id='select-multiple-chip' label='Category' />}
+          renderValue={(selected) => (
+            <Box sx={{ display: 'flex', flexWrap: 'wrap', gap: 0.5 }}>
+              {selected.map((value) => (
+                <Chip
+                  key={value}
+                  label={value}
+                  sx={{
+                    backgroundColor: colors.light[60],
+                    border: '1px solid ' + colors.light[80],
+                  }}
+                />
+              ))}
+            </Box>
+          )}
+          MenuProps={MenuProps}
+        >
+          {categories.map((category) => (
+            <MenuItem key={category} value={category}>
+              {category}
+            </MenuItem>
+          ))}
+        </Select>
+      </FormControl>
+      <TextField
+        label='Description'
+        value={textValue}
+        onChange={handleChangeText}
+        variant='outlined'
+      />
+      <FormControl>
+        <InputLabel id='select-option'>Wallet</InputLabel>
+        <Select
+          labelId='select-option'
+          id='simple-select-option'
+          value={selectedWallet}
+          onChange={handleChangeWalletOption}
+          input={<OutlinedInput id='select-single-option' label='Option' />}
+        >
+          <MenuItem value='Paypal'>Paypal</MenuItem>
+          <MenuItem value='Google Pay'>Google Pay</MenuItem>
+          <MenuItem value='Apple Pay'>Apple Pay</MenuItem>
+        </Select>
+      </FormControl>
+      <Button
+        onClick={() => setShowAttachmentComponent(!showAttachmentComponent)}
+        component='label'
+        sx={{
+          backgroundColor: colors.light[100],
+          border: `1px dashed ${colors.dark[25]}`,
+          color: colors.dark[50],
+        }}
+      >
+        <AttachFileIcon sx={{ marginRight: '10px' }}></AttachFileIcon>
+        Add attachment
+        {/* <input type='' hidden /> */}
+      </Button>
+      <StyledDiv>
+        <StyledDivRepeatSection>
+          <Typography
```

Comment:
All typography elements should be first created as shared components (https://www.figma.com/file/6KSlLFezGkD8uZXhuTsx3j/Montra---Expense-Tracker-UI-Kit-(Community)?type=design&node-id=216-2&mode=design&t=iiVfE1da8qtuIjHt-0).
Then, you just pick from shared components and use the one that is specified in the design.

Reply:
I don't have much figma experience so I didn't notice, know about know. I will try

Reply:
Anna bunu yaptım ancak oluşturduğum shared componenti olabildiğince customize edilebilir şekilde yaptım. Proplar alarak customize edilebiliyor. Sorun olmaz heralde.

Reply:
Responded here: https://github.com/frontend-from-0/montra-client/pull/8#discussion_r1594581758 :)